src/components/navigation-accordion/navigation-accordion.js

```diff
@@ -101,8 +101,15 @@ export default class NavigationAccordion extends React.Component {
         return page.path !== parentPage;
       })
       .map((page) => (
-        <li className="mb3" key={page.title}>
+        <li
+          className={classnames('mb3', {
+            'scroll-hidden': page.tag
```

Comment:
I believe the comment on lines 111-112 is meant to go here?

Reply:
Whoops! Thanks for catching that.